spring-boot-cli/src/test/java/org/springframework/boot/cli/command/init/InitCommandTests.java

```diff
@@ -292,6 +294,12 @@ public void parseProjectOptions() throws Exception {
 		assertEquals("grunt", this.handler.lastRequest.getBuild());
 		assertEquals("web", this.handler.lastRequest.getFormat());
 		assertEquals("ant-project", this.handler.lastRequest.getType());
+		assertEquals("groovy", this.handler.lastRequest.getLanguage());
+		assertEquals("org.test", this.handler.lastRequest.getGroupId());
+		assertEquals("demo", this.handler.lastRequest.getArtifactId());
+		assertEquals("demo", this.handler.lastRequest.getName());
+		assertEquals("0.0.1-SNAPSHOT", this.handler.lastRequest.getVersion());
```

Comment:
You should deviate from the service's default here otherwise you're not testing anything.